docs/lib/nyc-taxi.parquet.sh

```diff
@@ -0,0 +1,25 @@
+duckdb :memory: << EOF
```

Comment:
Should this have a shebang? One of these probably…

```suggestion
#!/usr/bin/env bash
```

```suggestion
#!/bin/sh
```

Reply:
I don't think so; shebang are only used for .exe?

Reply:
Ah, so even if you have a shebang, it’s ignored because we explicitly run it with `sh`?

Reply:
yes sh ignores the shebang and interprets the rest of the code directly

```sh
❯ chmod +x ./test.py
❯ cat ./test.py
#! /usr/bin/env python3
print("hello, world")
❯ bash ./test.py
./test.py: line 2: syntax error near unexpected token `"hello, world"'
./test.py: line 2: `print("hello, world")'
❯ ./test.py
hello, world
```

Reply:
Thanks for educating me. 😄